tarsier/tag_utils.ts

```diff
@@ -278,11 +283,14 @@ window.tagifyWebpage = (tagLeafTexts = false) => {
   const allElements = getAllElementsInAllFrames();
   const rawElementsToTag = getElementsToTag(allElements, tagLeafTexts);
   const elementsToTag = removeNestedTags(rawElementsToTag);
-  const idToXpath = insertTags(elementsToTag, tagLeafTexts);
+  const idToTagMeta = insertTags(elementsToTag, tagLeafTexts);
   shrinkCollidingTags();
   ensureMinimumTagFontSizes();
 
-  return idToXpath;
+  return Object.entries(idToTagMeta).reduce((acc, [id, meta]) => {
+    acc[parseInt(id)] = meta.xpath;
+    return acc ;
+  }, {} as { [key: number]: string });
```

Comment:
Just still returning the same id: xpath format @seanmcguire12. Basically the tarsier python should still return this for now, but prior to that we'll store the metadata of the text node index